docs_sphinx/usage/usage.rst

```diff
@@ -0,0 +1,122 @@
+Usage
```

Comment:
I#d rename this to "usage comparison" or something like that, otherwise it feels odd that under "Usage" 2/3 are from other packages. Even better would be to make it into a more general comparison and discuss things like whether it inherits from `ndarray` or not, how to convert into a standard numpy array and/or Python float, etc.